### PR TITLE
led_off function

### PR DESCRIPTION
**Pull request by BCKM**

### CODE REVIEW

**Comment by roboticsware** [comment]
If commit message is started with verb to explain what kinds of the behavior there, it'll be better. For example, "Add led_off function".

**Comment by BCKM** [comment]
> Thanks your PR. If commit message is started with verb to explain what kinds of the behavior there, it'll be better. For example, "Add led_off function".

OK

File: neopia/neosoco.py [hunk 1]
```diff
@@ -391,6 +391,25 @@ def get_value(self, port: str):
         else:
             raise TypeError
 
+    def led_off(self, port: str):
+        cvt_val = 0
```
Comment (roboticsware) [inline neopia/neosoco.py:395]: It's better not using cvt_val variable but, to insert the value, 0 into self.write() is more intuitive to understand codes. Second.

**Review by roboticsware — CHANGES_REQUESTED** [review]
Fix it mentioned and re-PR. Thanks.

**Comment by roboticsware** [comment]
And one more thing, there is always no Conflicts in your PR.

**Pull request closed without merging**